lib/ansible/modules/monitoring/zabbix_host.py

```diff
@@ -90,6 +91,44 @@
         required: false
         default: None
         version_added: '2.1'
+    ipmi:
```

Comment:
Zabbix API doesn't document ipmi as a dictionary, rather as separate objects (`ipmi_authtype`, `ipmi_privilege`, `ipmi_username` and `ipmi_password`). I would like see them as a separate options to this module with a note that at least username and password must be used together.

Options `ipmi_authtype` and `ipmi_privilege` will need defaults, e.g. `ipmi_privilege=dict(type='int', required=False, default=2)` and `version_added:` too.